pkg/sql/logictest/testdata/logic_test/exec_hash_join

```diff
@@ -0,0 +1,70 @@
+# LogicTest: 5node-dist 5node-dist-opt 5node-dist-metadata
```

Comment:
Remove these configs and replace with `local-vec`, which automatically sets experimental_vectorize like you did below.